### ISSUE TITLE
Slash key to opening chat

### ISSUE THREAD

**Issue by superJK92iscool** [SpoilerRules/KotlinizedMCP#10]
Go into a world and press the "/" key and noting happens.

**Comment by SpoilerRules** [comment]
You bound chat to the slash key, but couldn't open it?

**Comment by superJK92iscool** [comment]
No, it doesn't open it like it does in regular 1.8.9 (the Open Command keybind)

**Comment by SpoilerRules** [comment]
Are you using the release version, or did you clone the MCP branch?
